packages/dbus-broker/Cargo.toml

```diff
@@ -23,4 +23,4 @@ sha512 = "711e3db030cd8d3953e966c11d2afa4c543548f882d3364fc0b79b7cfb3b8d5ecb65b1
 glibc = { path = "../glibc" }
 libexpat = { path = "../libexpat" }
 libselinux = { path = "../libselinux" }
-systemd = { path = "../systemd" }
+systemd-252 = { path = "../systemd-252" }
```

Comment:
It'd be worth digging into each of these and determining the nature of the build-time dependency.

The [libsystemd docs](https://www.freedesktop.org/software/systemd/man/latest/libsystemd.html#Interface%20stability) say:
> Strict backwards-compatibility is maintained for the API (application programming interface) and ABI (application binary interface). Symbol versioning is used, with symbols only added and never removed.

If the dependency is purely on `libsystemd.so` then this seems OK.